hibernate-core/src/main/java/org/hibernate/query/sqm/mutation/internal/temptable/TableBasedInsertHandler.java

```diff
@@ -337,6 +337,40 @@ else if ( entityDescriptor.getGenerator() instanceof OptimizableGenerator ) {
 		);
 	}
 
+	// For Hibernate Reactive
```

Comment:
```suggestion
	/**
	 * For Hibernate Reactive
	 */
```

So it gets moved with any refactoring

Reply:
Thanks,  would it be OK  sense to add an annotation in the future?